MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/DataProcessorUI/GenericDataProcessorPresenter.h

```diff
@@ -109,19 +115,14 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS GenericDataProcessorPresenter
   DataProcessorMainPresenter *m_mainPresenter;
   // stores whether or not the table has changed since it was last saved
   bool m_tableDirty;
-  // Index for column 'Group'
-  int m_colGroup;
   // stores the user options for the presenter
   std::map<std::string, QVariant> m_options;
   // Post-process some rows
-  void postProcessGroup(int groupId, const std::set<int> &rows);
+  void postProcessGroup(const std::set<std::vector<std::string>> &data);
```

Comment:
OK. I would suggest making a proper type around this stuff at some point. I wouldn't have clue looking at that that that's the information it contained. I know that you inherited this, but might be nice to sort it out.